test/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
@makarandp0 Indent with 2 spaces. Also, extend the the root `.eslintrc.json` here.

Reply:
@makarandp0 Can you make sure you extend the root `.eslintrc.json` here?

Reply:
It's not needed, because file hierarchy automatically enforces this. (https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy) I have verified that rules from root are applied in this folder.